grails-app/domain/org/pih/warehouse/core/Location.groovy

```diff
@@ -242,6 +242,10 @@ class Location implements Comparable<Location>, java.io.Serializable {
         return locationType?.isFacilityLocation()
     }
 
+    Boolean isZoneLocation() {
```

Comment:
Let's just call this isZone() throughout.

Reply:
I know it's confusing because you might think it should be `isBin()` instead of `isBinLocation()` but bin location is the actual name for the thing and zone is the actual name for the thing.